engine/runtime/src/main/scala/org/enso/interpreter/instrument/command/CloseFileCmd.scala

```diff
@@ -17,13 +17,13 @@ class CloseFileCmd(request: Api.CloseFileNotification) extends Command(None) {
     ec: ExecutionContext
   ): Future[Unit] =
     Future {
-      ctx.locking.acquireFileLock(request.path)
       ctx.locking.acquireReadCompilationLock()
```

Comment:
In `acquireReadCompilationLock()` verify there `fileLocks.isEmpty()`

Reply:
Isn't that too restrictive? You could be dealing with multiple files at the same time.